### ISSUE TITLE
Problem with the CI Pipeline

### ISSUE THREAD

**Issue by LordRatte** [theletterf/english-lang#20]
I see the status is marked as stable but I'm using the latest version and I would say that it is anything but stable.

**Comment by Brawldude2** [comment]
Indeed. There are so many inconsistencies, how did this even pass tests?

**Comment by Sim3-14159** [comment]
I've noticed that, too. I've raised issue #22 on a specific aspect of this. Hopefully it helps a bit.
